gadget.Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN set -ex; \
 	export DEBIAN_FRONTEND=noninteractive; \
 	apt-get update && \
 	apt-get install -y gcc make golang-1.16 ca-certificates git clang \
-		software-properties-common && \
+		software-properties-common libseccomp-dev && \
```

Comment:
Is it required because libseccomp-golang requires it, right?

Reply:
Yes. This is only in the "builder" container.